### ISSUE TITLE
Origination Rates Import doesn't work in ASTPP 4.0

### ISSUE THREAD

**Issue by timur-v** [iNextrix/ASTPP#517]
Hello,
Tariff -> Origination Rates - > Import 
A correctly formed file is not accepted:
![image](https://user-images.githubusercontent.com/51515806/59150711-fd3ae900-89f5-11e9-8e01-abcb11f71fb9.png)
For that test I took the file downloaded from ASTPP itself ("Get sample file" section on the same page in ASTPP 4.0 demo).
Here is the test file content:
![image](https://user-images.githubusercontent.com/51515806/59150733-5acf3580-89f6-11e9-8022-718fd6d794bb.png)
Meantime in demo http://latest.astppbilling.org/rates/origination_rate_import/ this feature works fine.
My instance of ASTPP 4.0 is working on virtual machine.

**Comment by timur-v** [comment]
I found what causes this issue. It happens due to incorrect read/write permissions in /var/www/html/astpp/ directory. Owner of those files is root user, so web-server may not write new files there. ASTPP installation script consists this: "chown -Rf www-data.www-data /var/www/html/astpp", but /var/www/html/astpp/ is actually a symbolic link and chown doesn't change symbolic links by default. Files are actually placed in /opt/ASTPP/web_interface/astpp directory.
So, solution for this issue may be either "chown -LRf www-data.www-data /var/www/html/astpp" or "chown -Rf www-data.www-data /opt/ASTPP/web_interface/astpp".
I did this and was able to install addons then.
But I'd prefer to have developers' confirmation that such a solution doesn't create a breach in web server security.
